jenkins/ci.suse.de/cloud-mkcloud8.yaml

```diff
@@ -50,7 +50,7 @@
         - 'cloud-mkcloud{version}-job-ha-linuxbridge-{arch}'
         - 'cloud-mkcloud{version}-job-ha-ironic-{arch}'
         - 'cloud-mkcloud{version}-job-ha-ironic-agent-{arch}'
-        - 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-mariadb-{arch}'
```

Comment:
I might be missing something, but why does this require a completely separate job template? Is it because the different number of nodes? And if yes, wouldn't the non-mariadb job also require an additional node?

Reply:
Non-mariadb job is the one for 6-7 upgrade - that's using DRBD AFAIK.
Now I think the number of different parts for 6-7/7-8 scenario justifies extra job instead of tweaking the arguments.

Reply:
Hm, true it's using two controller nodes in the 6->7 case. So I guess I am fine with this.